internal/ioutil/print.go

```diff
@@ -52,3 +53,24 @@ func PrintCertificateMap(w io.Writer, v []config.CertificateReference) error {
 	}
 	return tw.Flush()
 }
+
+func PrintVerificationResults(w io.Writer, v []*verification.SignatureVerificationOutcome, resultErr error, digest string) error {
+	tw := newTabWriter(w)
+
+	if resultErr == nil {
+		fmt.Fprintf(tw, "Signature verification succeeded for %s\n", digest)
+		// TODO: print out failed validations as warnings.
+		return nil
+	}
+
+	fmt.Fprintf(tw, "ERROR: %s\n\n", resultErr.Error())
+	printOutcomes(tw, v, digest)
+
+	return tw.Flush()
+}
+
+func printOutcomes(tw *tabwriter.Writer, outcomes []*verification.SignatureVerificationOutcome, digest string) {
+	fmt.Printf("Signature verification failed for all the %d signatures associated with digest: %s\n", len(outcomes), digest)
+
```

Comment:
I understand that we are working on a spec to describe the CLI inputs and outputs, but we should print the error messages associated with the individual outcomes in the short-term to make this command more useful. Can you print the outcomes like below in the interim?

```
Signature verification failed for all the 3 signatures associated with digest: asdf....asdfasdf

Signature #1 : <Error>
Signature #2 : <Error>
Signature #3 : <Error>

```

Reply:
Good call, I'll add error messages.